apm-agent-plugins/apm-opentracing-plugin/src/main/java/co/elastic/apm/agent/opentracing/impl/ApmSpanBuilderInstrumentation.java

```diff
@@ -92,6 +93,7 @@ public static Object createSpan(@Advice.Argument(value = 0, typing = Assigner.Ty
                                                                 String operationName, long microseconds,
                                                                 @Nullable Iterable<Map.Entry<String, String>> baggage, ClassLoader applicationClassLoader) {
             AbstractSpan<?> result = null;
+            ElasticApmTracer tracer = GlobalTracer.getTracerImpl();
```

Comment:
Why can't we use `GlobalTracer.get()` here instead ?

Reply:
Because `Tracer#startSpan` is not part of the `Tracer` interface. That is because it returns a non-null span but because the `NoopTracer` only returns nulls, we can't add it to the interface. Normally it's not an issue because spans are started via `getActive().createSpan`